rt/frontend/jaxws/src/main/java/org/apache/cxf/jaxws/spi/GeneratedWrapperClassLoader.java

```diff
@@ -0,0 +1,106 @@
+import org.apache.cxf.jaxws.support.JaxWsServiceFactoryBean;
+import org.apache.cxf.service.model.InterfaceInfo;
+import org.apache.cxf.service.model.MessageInfo;
+import org.apache.cxf.service.model.MessagePartInfo;
+import org.apache.cxf.service.model.OperationInfo;
+import org.apache.cxf.wsdl.service.factory.ReflectionServiceFactoryBean;
+
+public class GeneratedWrapperClassLoader implements WrapperClassCreator {
+    private Set<Class<?>> wrapperBeans = new LinkedHashSet<>();
+    private InterfaceInfo interfaceInfo;
+    private JaxWsServiceFactoryBean factory;
+
+    public GeneratedWrapperClassLoader() {
+    }
+
+    @Override
+    public Set<Class<?>> generate(Bus bus, JaxWsServiceFactoryBean fact, InterfaceInfo ii, boolean q) {
+        factory = fact;
+        this.interfaceInfo = ii;
+        for (OperationInfo opInfo : interfaceInfo.getOperations()) {
+            if (opInfo.isUnwrappedCapable()) {
+                Method method = (Method)opInfo.getProperty(ReflectionServiceFactoryBean.METHOD);
+                if (method == null) {
+                    continue;
+                }
+                MessagePartInfo inf = opInfo.getInput().getFirstMessagePart();
+                if (inf.getTypeClass() == null) {
+                    MessageInfo messageInfo = opInfo.getUnwrappedOperation().getInput();
+                    createWrapperClass(inf,
+                            messageInfo,
+                            opInfo,
+                            method,
+                            true);
+                }
+                MessageInfo messageInfo = opInfo.getUnwrappedOperation().getOutput();
+                if (messageInfo != null) {
+                    inf = opInfo.getOutput().getFirstMessagePart();
+                    if (inf.getTypeClass() == null) {
+                        createWrapperClass(inf,
+                                messageInfo,
+                                opInfo,
+                                method,
+                                false);
+                    }
+                }
+            }
+        }
+        return wrapperBeans;
+    }
+
+    private void createWrapperClass(MessagePartInfo wrapperPart,
+                                    MessageInfo messageInfo,
+                                    OperationInfo op,
+                                    Method method,
+                                    boolean isRequest) {
+        boolean anonymous = factory.getAnonymousWrapperTypes();
+
+        String pkg = getPackageName(method) + ".jaxws_asm" + (anonymous ? "_an" : "");
+        String className = pkg + "."
+                + StringUtils.capitalize(op.getName().getLocalPart());
+        if (!isRequest) {
+            className = className + "Response";
+        }
+
+        Class<?> clz = null;
+        try {
+            clz = ClassLoaderUtils.loadClass(className, GeneratedWrapperClassLoader.class);
+        } catch (ClassNotFoundException e) {
```

Comment:
Probably we should log the exception here (debug or trace), just to help with troubleshooting